Src/FluentAssertions/Equivalency/EquivalencyAssertionOptions.cs

```diff
@@ -40,6 +39,17 @@ public EquivalencyAssertionOptions<TExpectation> Excluding(Expression<Func<TExpe
             return this;
         }
 
+        /// <summary>
+        /// Excludes the specified (nested) member from the structural equality check.
+        /// Allows to navigate deeper by using the returned <see cref="NestedExclusionOptionBuilder{TExpectation,TCurrent}"/>.
```

Comment:
🔧 I think it would be better to refer to the `ThenExcluding` member instead of the type. The former is what consumers will be using. The latter is an internal detail.

Reply:
✅ done